README.md

```diff
@@ -18,6 +18,7 @@ Options:
                                 to index all WARC records in the file
     --screenshot-mode           Special Wayback Machine mode for handling WARCs
                                 containing screenshots
+    --video-mode                Special Wayback Machine mode for handling WARCs containing video
```

Comment:
I believe `--video-mode` option is now gone. this line should be removed?

Reply:
Thought I did - agreed it should be removed.